LayoutTests/fast/events/message-port-clone-expected.txt

```diff
@@ -7,6 +7,6 @@ SUCCESS: Posting entangled port: DataCloneError: The object can not be cloned.
 SUCCESS: Posting cloned port.
 SUCCESS: Posted messages to cloned port.
 SUCCESS: Cloned both endpoints of a channel.
-SUCCESS: Got closed port with event message closed
+SUCCESS: Sending a closed port threw
```

Comment:
Would this test be a good candidate to promote to WPT?

Reply:
It is already tested by WPT.

Reply:
Should we remove the test, then, and help reduce bot burden?